TunnelVisionLabs.ReferenceAssemblyAnnotator/TunnelVisionLabs.ReferenceAssemblyAnnotator.targets

```diff
@@ -62,9 +62,12 @@
           DependsOnTargets="GetReferenceAssemblyPaths">
     <ItemGroup>
       <AvailableUnannotatedReferenceAssemblyDirectory Include="$(TargetFrameworkDirectory)" />
-      <!-- Use an intermediate item group so AvailableUnannotatedReferenceAssembly contains the expanded glob -->
-      <_AvailableUnannotatedReferenceAssembly Include="@(AvailableUnannotatedReferenceAssemblyDirectory->'%(Identity)*.dll')" />
-      <AvailableUnannotatedReferenceAssembly Include="@(_AvailableUnannotatedReferenceAssemblyDirectory)" />
```

Comment:
:question: was this supposed to use `_AvailableUnannotatedReferenceAssembly` instead of `_AvailableUnannotatedReferenceAssemblyDirectory`?

Reply:
I tried that first. It did not expand the wildcards, just copied the items exactly.

Reply:
I tried that first. It did not expand the wildcards, just copied the items exactly.